script/Arcueid/Day 7/07_00_ARC07_3_1_1.txt

```diff
@@ -28,9 +28,8 @@ I'll ask her about their relationship in more detail.
 [sha:d7ef14ac084f5315e8174003c2a6ecfe615a2742]{
 -- Page 522, Offset 17867.
 -- 「お、さっすが人力だけでピラミッドを建てちゃう霊長類！　堅実かつ誠実ね。
-// Avoiding direct use of 'human' in 人力 since we're discussing 'primates'
 // What are we trying to get at with 堅実 vs/plus 誠実
-"Wow, just what I'd expect from the primates that constructed the pyramids with nothing but their own two hands!
+"Wow, just what I'd expect from the Hominoids that constructed the pyramids with nothing but their human strength!
```

Comment:
is Hominoids supposed to be capitalized?

Reply:
I'm not sure to be honest. It's capitalized in the spreadsheet which was my reasoning

Reply:
hmm it doesnt say it needs to be capitalized on the next column